api/v1beta1/conditions.go

```diff
@@ -35,6 +35,9 @@ const (
 
 	// KeystoneServiceOSUserReadyCondition Status=True condition which indicates if the service user got created in the keystone instance is ready/was successful
 	KeystoneServiceOSUserReadyCondition condition.Type = "KeystoneServiceOSUserReady"
+
+	// KeystoneMemcachedReadyCondition - Indicates the Keystone memcached service is ready to be consumed by Keystone
+	KeystoneMemcachedReadyCondition condition.Type = "KeystoneMemcached"
```

Comment:
Perhaps the value should `KeystoneMemcachedReady` to keep with the `<ServiceName><Component>Ready` pattern?

Reply:
Yes, that makes sense. I'm fixing it in horizon-operator as well. https://github.com/openstack-k8s-operators/horizon-operator/pull/157